packages/utils/src/schema/getDefaultFormState.ts

```diff
@@ -98,7 +98,7 @@ export function computeDefaults<
   parentDefaults?: T,
   rootSchema: S = {} as S,
   rawFormData?: T,
-  includeUndefinedValues = false
+  includeUndefinedValues: boolean | "once" = false
```

Comment:
Also, update the function docs to add details for the new option
```suggestion
  includeUndefinedValues: boolean | "noRecursion" = false
```